src/staticanalysis/bot/static_analysis_bot/workflows/local.py

```diff
@@ -64,16 +67,42 @@ def clone(self):
                                     purge=True,
                                     sharebase=settings.repo_shared_dir,
                                     branch=b'central')
-
         cmd.insert(0, hglib.HGPATH)
-        proc = hglib.util.popen(cmd)
```

Comment:
Nit: Maybe we should keep this, and output stdout and stderr separately?
`clone: {}` for stdout
`clone (err): {}` for stderr